src/gpg/contact/export.py

```diff
@@ -6,11 +6,16 @@
 def export(export_path, homedir, contacts):
     c = gpg.Context(armor=True, home_dir=homedir)
 
-    if(export_path == "-"):
+    if export_path == "-":
         export_file = sys.stdout
     else:
-        export_file = open(export_path, "w")
-
+        try:
+            export_file = open(export_path, "w")
+        except PermissionError:
+            print("Not enough permissions to write to {path}".format(
+                path=export_path
+            ), file=sys.stderr, flush=True)
+            exit(1)
```

Comment:
These lines seem to be the same everywhere. Why don't you simplify them with a utility function like this: 
```
print_error("Not enough permissions to write to {path}".format(path=export_path))
```